app/lib/navigation/state_file_nj_question_navigation.rb

```diff
@@ -2,11 +2,16 @@ module Navigation
   class StateFileNjQuestionNavigation < Navigation::StateFileBaseQuestionNavigation
     include ControllerNavigation
 
+    def self.show_progress?(controller_class)
+      section = get_section(controller_class)
+      section && section.increment_step?
+    end
+
+    section_4_title = I18n.t("state_file.navigation.nj.section_4", filing_year: MultiTenantService.statefile.current_tax_year)
```

Comment:
[dust] Going to make a weak argument for inlining this even though it'll make the line much longer. Feels strange to call out just one section variable up here, it made me pause and go through trying to figure out what was so special about section 4.

Reply:
that's true, it is weird. i'll get more creative with indenting and do it in-line 👍

Reply:
done - and didn't even need to indent since other lines are longer anyway